cmd/root.go

```diff
@@ -64,23 +64,26 @@ func run(testFileGlobs []string) error {
 
 	if len(allTestCases) == 0 {
 		fmt.Println("WARNING: No tests discovered. Exiting...")
-		return nil
+		return 0, err
 	}
 
 	testRunner := pat.GoTestRunner{}
 	retCode := testRunner.RunTests(allTestCases)
 	if retCode != 0 {
-		return errors.New("Bad return code from test")
+		return retCode, errors.New("Bad return code from test")
 	}
-	return nil
+	return 0, nil
 }
 
 // RootCmd represents the base command when called without any subcommands
 var RootCmd = &cobra.Command{
 	Use:   "pat <test-file-globs...>",
 	Short: "Prometheus Alert Testing utility",
 	Run: func(cmd *cobra.Command, args []string) {
-		run(args)
+		if code, err := run(args); err != nil {
+			fmt.Printf("exited with %s", strconv.Itoa(code))
```

Comment:
Why not use `%d` and get rid of `Itoa` call?

Reply:
That's a pretty good idea :-)

Change pushed.